nodebuilder/tests/swamp/config.go

```diff
@@ -35,3 +37,20 @@ func WithBlockTime(t time.Duration) Option {
 		c.Tendermint.Consensus.SkipTimeoutCommit = false
 	}
 }
+
+// WithLogLevel overrides basic log levels for test purposes
+func WithLogLevel(components []string, levels []string) Option {
```

Comment:
How about make it 
```
func WithLogLevel(level string, components ...string) Option
```
So log level could be applied to multiple components. 
Multiple levels could be applied with several WithLogLevel calls with corresponding levels. 
This could simplify logic and usage.

Reply:
Yeap, that makes a lot of thanks. Thank you. I'll rework it